stackdriver_exporter.go

```diff
@@ -196,7 +190,9 @@ func main() {
 	}
 
 	projectIDs := strings.Split(*projectID, ",")
-	handler := newHandler(projectIDs, monitoringService, logger)
+	metricsTypePrefixes := strings.Split(*monitoringMetricsTypePrefixes, ",")
+	metricExtraFilters := parseMetricExtraFilters()
+	handler := newHandler(projectIDs, metricsTypePrefixes, metricExtraFilters, monitoringService, logger)
```

Comment:
Nit: I like being explicit about params required vs depending on values further down the rest of the file seems to reference the global flags directly. Not sure if it might get brought up when PR'ing upstream.

Reply:
You mean the whole PR, or just passing paramaters around vs. referencing global ones in this "main" file?